docs/developers_guide/ocean/test_groups/single_column.md

```diff
@@ -43,6 +51,9 @@ The class {py:class}`polaris.ocean.tests.single_column.viz.Viz`
 produces figures comparing the initial and final profiles of temperature and
 salinity.
 
+For the ideal age tracer case, an additional figure is produced comparing the 
+initial and final profile of ideal age.
+
```

Comment:
I think with my comments below, nothing specific to ideal age woulds be needed.

Reply:
removed